Makefile

```diff
@@ -529,6 +529,13 @@ check_third_party_binary_for_engine:
 
 check_engine_integration_test:
 	./engine/test/utils/check_case.sh
+	./engine/test/integration_tests/run_group.sh "check others"
+
+check_dm_integration_test:
+	./dm/tests/run_group.sh "check others"
+
+check_cdc_integration_test:
+	./tests/integration_tests/run_group.sh check "others"
```

Comment:
Why don't we unify it to `"check others"` or `check "others"`?

Reply:
Because `./tests/integration_tests/run_group.sh` and `./dm/tests/run_group.sh` have a different number of parameters.

Reply:
The main reason is that there are ci scripting issues involved here. No matter how you change it, there will always be inconsistent behavior, either inside the script or inside the makefile.